nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -100,6 +111,61 @@ public void shouldFailWhenMovingToColdStorageDocumentWithoutContent() {
         }
     }
 
+    @Test
+    public void shouldRequestRetrievalDocumentBlobFromColdStorage() {
```

Comment:
Maybe use a simpler name for readability as we know we're in the TestColdStorage class: "shouldRequestRetrieval"?